Cargo.toml

```diff
@@ -7,9 +7,8 @@ authors = ["Polygon Avail Team"]
 # See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
 
 [dependencies]
-kate-recovery = { git = "https://github.com/availproject/avail-core.git", tag = "kate-recovery/v0.8.1" }
-avail-subxt = { git = "https://github.com/availproject/avail.git", tag = "v1.5.0" }
-subxt = "0.27.1"
+kate-recovery = { version = "0.8", git="https://github.com/availproject/avail-core", tag = "da-primitives/v0.4.6" }
+avail-subxt = { path = "../avail_1/avail-subxt" }
```

Comment:
Is this intentional?

Reply:
Yes, I was using this meanwhile the `avail-core` PR was in progress. 
Now it is fixed and pointing to a proper tagged version.